AsyncDisplayKit/ASMultiplexImageNode.h

```diff
@@ -195,7 +195,7 @@ didFinishDownloadingImageWithIdentifier:(id)imageIdentifier
  * @abstract An image URL for the specified identifier.
  * @param imageNode The sender.
  * @param imageIdentifier The identifier for the image that will be downloaded.
- * @discussion Supported URLs include assets-library, Photo framework URLs (ph://), HTTP, HTTPS, and FTP URLs.  If the
+ * @discussion Supported URLs include assets-library, URLs converted from ASPhotosImageRequest, HTTP, HTTPS, and FTP URLs.  If the
```

Comment:
Let's improve this comment, now that a client would have to use the new class manually.  Probably break it up into two lines, and expand into a sentence like "Photos.framework URLs (ph://) may be provided to an ASPhotosImageRequest, which can generate a URL to return here" ?

Have you considered making ASMultiplexImageNode create the ASPhotosImageRequest internally by parsing the ph:// URLs?  Particularly because the ASPhotosImageRequest object can't be passed to Multiplex directly, this API feels odd to me (I may be misunderstanding it).